src/database/mongodb.ts

```
@@ -0,0 +1,12 @@
+import mongoose from 'mongoose';
+
+export default async function connect() {
+  try {
+    await mongoose.connect(process.env.MONGODB || '');
+  } catch (error) {
+    throw new Error(
+      'Please define the MONGODB environment variable inside .env. Error: ' +
+        error
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)